README.md

```diff
@@ -1,37 +1,216 @@
+```
+
+#### Statistics Aggregator
+
+```bash
+docker run --name statistics cccs/assemblyline-core python -m assemblyline_core.metrics.run_statistics_aggregator
+```
+
+### Scaler
 
 Spin up and down services in the system depending on the load.
 
-##### Updater
+```bash
+docker run --name scaler cccs/assemblyline-core python -m assemblyline_core.scaler.run_scaler
+```
+
+### Updater
 
 Make sure the different services get their latest update files.
 
-##### Workflow
+```bash
+docker run --name updater cccs/assemblyline-core python -m assemblyline_core.updater.run_updater
+```
+
+### Workflow
 
 Run the different workflows in the system and apply their labels, priority and status.
+
+```bash
+docker run --name workflow cccs/assemblyline-core python -m assemblyline_core.workflow.run_workflow
+```
+
+## Documentation
+
+For more information about these Assemblyline components, follow this [overview](https://cybercentrecanada.github.io/assemblyline4_docs/overview/architecture/) of the system's architecture.
+
+---
+
+# Assemblyline 4 - Core
+
+Ce dépôt fournit des services de base pour Assemblyline 4.
+
+## Variantes d'images et balises
+
+| **Type d'étiquette** | **Description**                                                                                                                    |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                                   |          `latest`          |
+|      build_type      | Le type de compilation utilisé. `dev` est la dernière version instable. `stable` est la dernière version stable. `stable` ou `dev` |     `stable` ou `dev`      |
+|        series        | Le type de build utilisé. `dev` est le dernier build unstable : `version.buildType`.                                               | `4.5.stable`, `4.5.1.dev3` |
+
+## Composants
+
+### Alerter
+
+Créer des alertes pour les différentes soumissions dans le système.
+
+```bash
+docker run --name alerter cccs/assemblyline-core python -m assemblyline_core.alerter.run_alerter
+```
+
+### Archiver
+
+Archivage des soumissions, de leurs résultats et des fichiers dans l'archive.
+
+```bash
+docker run --name archiver cccs/assemblyline-core python -m assemblyline_core.archiver.run_archiver
+```
+
+### Dispatcher
+
+Achemine les fichiers dans le système pendant qu'une soumission est en cours. Assurez-vous que tous les fichiers pendant une soumission sont complétés par tous les services requis.
+
+```bash
+docker run --name dispatcher cccs/assemblyline-core python -m assemblyline_core.dispatching
+```
+
+### Expiration
+
+Supprimer les soumissions et leurs résultats lorsque leur TTL expire.
```

Comment:
"Supprime les soumissions et leurs résultats lorsqu'ils expirent." First of all the tense used for all the components should be consistent for visual and refering to an english term "TTL" is probably worth omiting ?